tests/test_anitopy.py

```diff
@@ -3,7 +3,7 @@
 from unittest import TestCase
 
 import anitopy
-from tests.fixtures.table import table
```

Comment:
The tests fail for python 3 because of the removal of `tests.` on the imports. You could keep it and it would still works on python 2.7 if you used the [`discover` argument to run unittest](https://docs.python.org/2/library/unittest.html#test-discovery).
With this little fix, python 3 passes all tests, but python 2 fails in some, all because of unicode problems. To fix this you just have to import `unicode_literals` from `__future__` in all files (except `__init__.py`) to convert all strings to unicode. [As demonstrated here.](http://python-future.org/compatible_idioms.html#unicode-text-string-literals)